sequence_tools.py

```diff
@@ -0,0 +1,145 @@
+# Importing modules
+from typing import Union
+import source.fastq_read as fr
+import source.protein_tools as pt
+import source.nucleic_acids_tools as nat
```

Comment:
По PEP8 импорты из стандартной библиотеки и импорты своих модулей разделяются пустой строкой
```suggestion
from typing import Union

import source.fastq_read as fr
import source.protein_tools as pt
import source.nucleic_acids_tools as nat
```